index.html

```diff
@@ -0,0 +1,215 @@
+                            </div>
+                        </div>
+
+                    </div>
+
+                </nav>
+            </header>
+            <section class="hero">
+                <div class="headline">
+                    <p>Your help goes a long way!</p>
+                    <h1 class="title-h1">Help us reach needy families and
+                        empower them get
+                        access to education. </h1>
+                </div>
+
+                <div class="card">
+                    <p class="headline-text">We aim to shape possibilities of
+                        under privileged
+                        children by providing a stepping stone to acquiring
+                        education and better living conditions.</p>
+                </div>
+
+                <q>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </q>
+
+            </section>
+
+        </div>
+        <section class="values">
+            <h3>Our Values</h3>
+            <hr>
+            <div class="values-container">
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li><img
+                                src="/first-capstone-project/assets/images/integrity.png" /></li>
+                        <li><p>Integrity</p>
+                        </li>
+                        <li>
+                            <p>We focus on the entire customer lifecycle.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="values-reasons" id="values">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/responsive.png" />
+                            <li><p>Responsive</p></li>
+                            <li>
+                                <p>To the needs of the children in our target
+                                    group
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/enriching.png" />
+                            <li><p>Enriching</p></li>
+                            <li>
+                                <p>
+                                    bettering lives through education and
+                                    other avenues
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+            </div>
+            <button type="button" class="donate-btn">Donate</button>
```

Comment:
kindly make this button appear on mobile size only and display a text on screen desktop  refer to the images below ,it's very important to stick to the design 
**design mobile**
![Capture d’écran du 2023-08-02 00-33-12](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/76afe306-d319-43ef-9342-5a681160851f)

**design desktop**
![Capture d’écran du 2023-08-02 00-32-52](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/8f23edd5-7c90-4551-a580-4a06a37dca1e)

**your desktop version**
![Capture d’écran du 2023-08-02 00-32-37](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/03e3ccc7-2412-4ebc-9222-8a8571594cdf)